content/docs/2.7/scalers/azure-app-insights.md

```diff
@@ -40,6 +46,9 @@ for further details.
 - `activeDirectoryClientId` - Id of the Active Directory client. The client must have `Monitoring Reader` permissions for the Application Insights instance.
 - `activeDirectoryClientPassword` - Password of the Active Directory client password.
 - `metricFilter` - Further specify the metrics query using a filter. For example `cloud/roleName eq 'example`. (Optional)
+- `cloud` - Name of the cloud environment that the Event Hub belongs to. (Values: `AzurePublicCloud`, `AzureUSGovernmentCloud`, `AzureChinaCloud`, `Private`, Default: `AzurePublicCloud`)
```

Comment:
Please, add `Optional` if the field is optional to follow the pattern
- name - Description. (Values: x, y, z, Default: y, Optional, Extra Info)

Reply:
Also added it for the EventHub scaler as I missed it there.